presto-main/src/main/java/com/facebook/presto/operator/aggregation/fixedhistogram/FixedDoubleHistogram.java

```diff
@@ -111,34 +130,31 @@ public int getRequiredBytesForSerialization()
 
     public void serialize(SliceOutput out)
     {
-        out.appendByte(FORMAT_TAG)
-            .appendInt(bucketCount)
-            .appendDouble(min)
-            .appendDouble(max)
-            .appendBytes(Slices.wrappedDoubleArray(weights, 0, bucketCount));
+        out
+                .appendInt(bucketCount)
```

Comment:
move this line together with previous one `out.appendInt(bucketCount)`

Reply:
Sorry, I don't understand this comment.  It also suggests changes from 117-118 to 133-134, and that doesn't seem to match my sources.

Reply:
The current source shows
```java
out
        .appendInt(bucketCount)
        .appendDouble(min)
        .appendDouble(max)
        .appendBytes(Slices.wrappedDoubleArray(weights, 0, bucketCount));
```
We usually do
```java
out.appendInt(bucketCount)
        .appendDouble(min)
        .appendDouble(max)
        .appendBytes(Slices.wrappedDoubleArray(weights, 0, bucketCount));